src/state/state.constants.ts

```diff
@@ -0,0 +1,31 @@
+import { DeepPartial } from 'redux';
+import uuid from 'uuid/v4';
+import { Environment } from '../common/types/environment.type';
+import { AppState } from './index';
+
+// Define a preloadedState for createStore()
+// See: https://redux.js.org/recipes/structuring-reducers/initializing-state
+
+export const DEFAULT_STATE: { [key in Environment]: DeepPartial<AppState> } = {
+    [Environment.Production]: {},
```

Comment:
See [this](https://github.com/decrn/localsurvey/pull/23#discussion_r254429033)

Reply:
> What if in production, when the user first loads the app, we suggest to load an example survey. Like as a modal pop-up, or a notification or something. We could use it as a "tour" of how you can build complex surveys and whatnot. Or even just provide it as a default survey. Deleting it is one click away, anyway.

Combining "show the real product" and "provide a demo" on GH-pages.

Reply:
I feel like it's out of scope for now, wanna solve this in the future instead since we're not set up properly regarding localstorage etc